test/fixtures/orderbook.js

```diff
@@ -372,6 +372,35 @@ module.exports.QUALITY_OFFERS = [
   }
 ];
 
+module.exports.INCORRECT_TAKER_PAYS_FUNDED_OFFERS = [
```

Comment:
Hmmm, I guess the taker pays funded result used to be incorrect for this offer when it was passed through `setOffers`. Any suggestions on what to call it?